From 58a9a4e2fda48c9b79749b2ebc8c797b424756e0 Mon Sep 17 00:00:00 2001
From: "Gogineni, GiribabuX" <giribabux.gogineni@intel.com>
Date: Fri, 27 Oct 2017 19:30:32 +0530
Subject: [PATCH 1073/1240] ASoC: Intel: Skylake: Add ULL machine driver entry

This patch adds ULL machine driver name to driver
data.

Change-Id: Ib6e47b14f7e7c8a24a3bdad4034a39008de7f3be
Signed-off-by: Gogineni, GiribabuX <giribabux.gogineni@intel.com>
Reviewed-by: Tewani, Pradeep D <pradeep.d.tewani@intel.com>
Reviewed-by: Shaik, Kareem M <kareem.m.shaik@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Reviewed-by: Kale, Sanyog R <sanyog.r.kale@intel.com>
Reviewed-by: audio_build <audio_build@intel.com>
Tested-by: Sm, Bhadur A <bhadur.a.sm@intel.com>
---
 sound/soc/intel/skylake/skl.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index 5fefb53..e2f15e8 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -1072,6 +1072,11 @@ static struct snd_soc_acpi_mach sst_bxtp_devdata[] = {
 		.drv_name = "bxt_tdf8532",
 		.fw_filename = "intel/dsp_fw_bxtn.bin",
 	},
+	{
+		.id = "INT34C3",
+		.drv_name = "bxt_ivi_ull",
+		.fw_filename = "intel/dsp_fw_ull_bxtn.bin",
+	},
 	{}
 };
 
-- 
2.7.4

